### PR TITLE
Fix measures keys

### PR DESCRIPTION
**Pull request by samanehsan**
- Switch measures questions 3 and 4 since their order was switched in the survey
- Remove measures question 17 and re-name measure 18 to 17
- Make `format_json.py` add the conditions to measures.questions.3 (previously measures.questions.4)

### CODE REVIEW

File: app/locales/en-us/translations.js [hunk 1]
```diff
@@ -280,7 +277,7 @@ const translations = {
                         "label": "I react annoyed if another person steals the show from me."
                     }
                 },
-                "label": "15. Please rate the extent to which you agree or disagree with the following statements:",
+                "label": "16. Please rate the extent to which you agree or disagree with the following statements:",
```
Comment (samchrisinger) [inline app/locales/en-us/translations.js]: Wondering if it makes more sense to just use the [`{{index}} variable`](http://stackoverflow.com/questions/19795562/accessing-index-in-each-in-emberjs) available in the `#each` context? That way if these are every reordered this doesn't have to be rewritten.

Comment (samanehsan) [reply app/locales/en-us/translations.js]: The `{{index}}` starts at zero so I would need to write a helper to get index + 1 -- does that sound like an okay approach @samchrisinger?

Comment (samchrisinger) [reply app/locales/en-us/translations.js]: Seems fine to me.

**Comment by samanehsan** [comment]
Done making that change @samchrisinger! The template now adds in the question numbers in this PR https://github.com/CenterForOpenScience/exp-addons/pull/119 😎

**Pull request merged**